content/vault/v1.20.x/content/docs/deploy/kubernetes/comparisons.mdx

```diff
@@ -0,0 +1,136 @@
+
+**Best for:** Organizations using multiple secret stores in addition to Vault, or those who prefer vendor-neutral CSI standards.
+
+## Vault Agent Injector
+
+The [Vault Agent Injector](/vault/docs/platform/k8s/injector) injects Vault Agent sidecar containers into pods. 
+The Agent containers authenticate with Vault and render secrets to a shared memory volume for consumption by application containers.
+
+**Best for:** Applications that want to reference multiple Vault secrets in one template, or need a wider array of auth methods.
+
+
+## Use case recommendations
+
+The appropriate integration for your environment depends on the applicable
+security policies, secret data governance, ease of use requirements, and
+system availability guarantees.
+
+| Limitation                                        | Vault Secrets Operator            | Vault Secrets Store CSI provider　| Vault Agent Injector
+| ------------------------------------------------- | --------------------------------- | ---------------------------- | --------------
+| Requires privileged pod access                    | No, except with CSI driver        | Yes                          | No
+| Flexibility in how to consume secrets             | Yes                               | Yes                          | No
+| Low resource consumption (I/O, CPU, Memory)       | Yes                               | Yes                          | No
+| Secret data limited to ephemeral volumes          | No                                | No                           | Yes
+| Pod Autoscaling dependent on Vault availability   | No                                | Yes                          | Yes
+| Personas easily separated via K8s [RBAC] policies | Yes                               | No                           | No
+
+[RBAC]: https://kubernetes.io/docs/reference/access-authn-authz/rbac
+
+
+## Performance considerations
+
+Performance characteristics vary significantly between integrations based on
+their architectural patterns.
+
+<Tabs>
+	
+<Tab heading="Load on Vault">
+
+- **Vault Secrets Operator** - Lowest load due to per-CRD connections and cluster-local caching.
+- **Vault Secrets Store CSI provider** - Higher load due to per-pod connections.
+- **Vault Agent Injector** - Highest load due to per-pod connections and sidecar pattern.
+	
+</Tab>
+	
+<Tab heading="Rendered secret lifecycle">
+
+- **Vault Secrets Operator** - Secret lifecycle is independent of pod lifecycle due 
+to caching in Kubernetes Secrets, unless using the Vault Secrets Operator CSI driver.
+- **Vault Secrets Store CSI provider** - Secret lifecycle depends on pod lifecycle, even with the 
+option to cache in Kubernetes Secrets (they will be deleted after the pod terminates).
+- **Vault Agent Injector** - Secret lifecycle depends on pod lifecycle.
+
+</Tab>
+	
+<Tab heading="Resource consumption">
+
+- **Vault Secrets Operator** - Lower consumption due to one client process needed per cluster.
```

Comment:
In the case of CSI it is one driver instance per Node

Reply:
Yeah let me add that back in; I'd mentioned that when VSO-CSI was its own bullet point but now we're discussing it as just part of VSO.